gui/src/components/Notifications.js

```diff
@@ -23,16 +23,24 @@ export class Notifications extends React.Component {
   }
 
   successPresenter = (success, i) => {
-    // TODO: Update backend to have JSONAPI responses on Create to better discern between types.
     const isJob = success => success.initiators
```

Comment:
Let's improve the `const isJob = success => success.initiators` check as well.

Reply:
@dimroc What should the condition be? I don't see anything unique for the Job in the response

Reply:
Great question @NavyAdmiral. I've submitted another PR so that the job response is consistent with the job run and bridge type response in that it has a `data.type`: https://github.com/NavyAdmiral/chainlink/pull/2

You should be able to now do something like the following:

```js
const isJob = success => success.data && success.data.type === 'jobs'
```

Reply:
@dimroc is it 'jobs' or 'specs'?

Reply:
Probably `specs`. Go with what you see you in the response.

Reply:
@dimroc it's specs as is in the commit